net/ola/Portfile

```diff
@@ -22,23 +21,10 @@ long_description    The Open Lighting Architecture (OLA) provides a plugin \
 
 homepage            http://www.openlighting.org/ola/
 
-checksums           rmd160  1312a43f8db8f211f68cd1ab390eb9c8bf476144 \
-                    sha256  8a65242d95e0622a3553df498e0db323a13e99eeb1accc63a8a2ca8913ab31a0 \
-                    size    2512577
```

Comment:
The checksums line needs to be restored and updated for the new version.

Reply:
Done, after I found how to generate your rmd160 hash on your docs.